astro.config.mjs

```diff
@@ -27,6 +29,16 @@ export default defineConfig({
         }
       }],
     ]
-  })],
+  }), icon()],
   site: 'https://ubie-vitals.github.io',
-});
\ No newline at end of file
+  vite: {
+    ssr: {
+      noExternal: ['@ubie/ubie-icons', '@ubie/ubie-ui']
```

Comment:
The reason why '@ubie/ubie-icons' and '@ubie/ubie-ui' did not work
`client:only` is no longer needed and display is faster.